airbyte-integrations/connectors/source-hardcoded-records/source_hardcoded_records/streams.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+
+from abc import ABC
+from typing import Any, Iterable, Mapping
+
+from airbyte_cdk.sources.streams import Stream
+
+
+class HardcodedStream(Stream, ABC):
```

Comment:
I would like to see a stream with 24 million rows of exactly this schemas and values:
```
{ field1: "valuevaluevaluevaluevalue1", 
field2: "valuevaluevaluevaluevalue1", 
field3: "valuevaluevaluevaluevalue1", 
field4: "valuevaluevaluevaluevalue1",
field5: "valuevaluevaluevaluevalue1"
}
```

this is the format we use for all of our other performance testing. doing the exact same thing helps make it comparable to other tests

Reply:
added

Reply:
makes sense. thank you!